### DISCUSSION TITLE
Flakey Presence Detection in Windows Sign-In

### DISCUSSION THREAD

**Discussion by kins-dev** [solokeys/solo2#116] [Q&A]
I've seen this with both the USB-A and USB-C keys. If I go to manage, and have a key already inserted, it will go to the dim LED state, (red+green). I can then touch it, watch it go full green, but Windows won't catch that I took an action. After 10-15 seconds the LED goes back dim. I have to remove and plug in the key multiple times for it to work.

Here's the fun part. Sometimes when doing this, it will jump directly to the management screen without me touching the key. It is almost like a command was queued up in a FIFO but it isn't flushed properly.

Anyone else experiencing this? Am I doing something wrong?

**Comment by weiss-johannes** [comment]
Exactly the same with me

**Comment by alexjmoore** [comment]
Similar, I have found if I touch the key immediately after windows prompts then it seems to somewhat reliably take it, however if I'm more than a second or two later to touch the key, it doesn't recognise it.  Running 1.0.9.  Also agree the LED colour change is almost imperceptible, would be good to change colour or flash.

I have a Solo1 that doesn't exhibit this, it changes colour to prompt a press and I can take as long as I want to press the button.

**Reply by AABatteries** [reply]
same here @alexjmoore. Couldn't figure out why Windows wouldn't accept me touching the key. It only works if I touch it immediately when plugging in
